edenia/proton-affiliate
# Fix/add translation to spanish

### Add translation to spanish

### What does this PR do?

- Resolve #404 

### Steps to test

1. Run the project locally
2. Configure to display the web in Spanish
3. Check that the texts are correct

### Screenshots

### **Home page**

![imagen](https://user-images.githubusercontent.com/66583677/219214640-1778018f-24e3-480f-8d74-2cdaac8b3d6e.png)

### **Footer**

![imagen](https://user-images.githubusercontent.com/66583677/219215337-23ba77f0-435c-4f9d-aab7-3f2b0513ba45.png)


### **Help page**

![imagen](https://user-images.githubusercontent.com/66583677/219214743-197445fb-7fed-48af-808f-a69c871fd1b7.png)

### **About page**

![imagen](https://user-images.githubusercontent.com/66583677/219214875-42f641a8-7d9e-4140-af6c-31129e8af49f.png)
![imagen](https://user-images.githubusercontent.com/66583677/219216425-8bf15dd9-ad89-4ac9-9075-2c814f55da3e.png)

### **Adminpage**

![imagen](https://user-images.githubusercontent.com/66583677/219215226-68b4364f-5f75-49b5-a329-a0370cde9547.png)

#### CheckList
